pkg/chart/mock.go

```diff
@@ -0,0 +1,40 @@
+package chart
+
+import (
+	"context"
+
+	"github.com/nais/knorten/pkg/database/gensql"
+	"github.com/nais/knorten/pkg/logger"
+)
+
+type ChartClientMock struct {
```

Comment:
Tror kanskje jeg bare hadde kalt denne `ClientMock`, for man kaller på den med pakkenavn.